frontend/app/(withCommonLayout)/[...not-found]/page.tsx

```
@@ -0,0 +1,145 @@
+                                        />
+                                        <path
+                                            d='M319.414 84.4042C318.096 84.4042 316.831 83.6391 316.277 82.3462C316.198 82.1615 308.265 64.3519 295.244 61.4496C288.945 60.0512 282.197 62.4258 275.212 68.5735C273.789 69.8135 271.628 69.6816 270.389 68.2568C269.15 66.8321 269.282 64.6685 270.705 63.4285C279.43 55.7769 288.207 52.8746 296.773 54.8007C313.194 58.4945 322.208 78.7843 322.577 79.6286C323.341 81.37 322.551 83.3752 320.811 84.1404C320.337 84.2987 319.862 84.4042 319.414 84.4042Z'
+                                            fill='#161439'
+                                        />
+                                        <svg
+                                            viewBox='0 0 649 323'
+                                            fill='none'
+                                            xmlns='http://www.w3.org/2000/svg'
+                                            width='100%'
+                                            height='100%'
+                                        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)